extensions/shared/clipboard-input.scss

```diff
@@ -1,7 +1,12 @@
 .jetpack-clipboard-input {
 	display: flex;
 
+	.components-text-control__input,
 	.components-clipboard-button {
-		margin: 2px 0 0 6px;
+		min-height: 36px;
```

Comment:
Since this is actually a Gutenberg core variable (`$button-size`), let's pull it as a variable in already now — that way we'll avoid fixes like this in future when things just get updated automatically:

https://github.com/WordPress/gutenberg/blob/bbc2390d5bd48d754e3cc4033d972eb2aa725e2e/packages/base-styles/_variables.scss#L39

You can have those variables available here by importing this:
```scss
@import './styles/gutenberg-base-styles.scss';
```

Reply:
Oh looks like it's actually pretty new value: https://github.com/WordPress/gutenberg/pull/20529 and isn't pushed to NPM yet. 😁 

So let's use the old `$icon-button-size` variable meanwhile? Jetpack has base-styles v1.0.0 (latest is v1.4.0) and both have that: 

https://unpkg.com/browse/@wordpress/base-styles@1.0.0/_variables.scss
https://unpkg.com/browse/@wordpress/base-styles@1.4.0/_variables.scss

The variable just gets updated whenever someone updates the package to the latest in Jetpack.

Reply:
> The variable just gets updated whenever someone updates the package to the latest in Jetpack.

Noting that this is tracked (update to 1.4.0) in #13593

Reply:
We're now running @wordpress/base-styles 1.4.0 as of #15059. You should be all set to use the variable now!

Reply:
I tried updating this and if I rebase to master and do an yarn install it shows @wordpress/base-styles  as 1.4.0 but _variables.scss still looks like the 1.0.0 versions - tried deleting node modules and yarn install again and go the same thing - any ideas why this might happen?

Reply:
It looks like `$button-size` was added on March 2. in https://github.com/WordPress/gutenberg/commit/7866ae1b48d8c3fd3c898598b6eac18b9fc2abe9#diff-0e7bf8af1d20999faf370cb9696b8c02 , but 1.4.0 was tagged on February 10. I'm afraid we'll need to wait for another release.

Until then, we can do what @simison suggested above maybe?

> So let's use the old $icon-button-size variable meanwhile?

Reply:
@gziolo would it be possible to tag latest version for `base-styles`? Would find it handy in Calypso too.

Reply:
WordPress 5.4 is currently scheduled to land on March 31 2020 and it's today :)

We can publish the latest version of all packages soon after, I personally can tackle it tomorrow if everything goes smooth.

Reply:
@jorgefilipecosta is handling the release later today.

Reply:
I added a PR to update to the latest version of base styles here: https://github.com/Automattic/jetpack/pull/15280

Reply:
done in ee61e1464f515cacbd88198949ffd3230392c664